service/src/test/java/bio/terra/cbas/dependencies/leonardo/TestAppUtils.java

```diff
@@ -434,5 +465,77 @@ public TestAppUtils() throws IOException {
                     "labels": {}
                 }""",
                 Map.of("workspaceId", workspaceId)));
+
+    cromwellRunnerAppResponse =
+        ListAppResponse.fromJson(
+            StringSubstitutor.replace(
+                """
+                {
+                    "workspaceId": "${workspaceId}",
+                    "cloudContext": {
+                        "cloudProvider": "AZURE",
+                        "cloudResource": "blah-blah-blah"
+                    },
+                    "kubernetesRuntimeConfig": {
+                        "numNodes": 1,
+                        "machineType": "Standard_A2_v2",
+                        "autoscalingEnabled": false
+                    },
+                    "errors": [],
+                    "status": "RUNNING",
+                    "proxyUrls": {
+                        "cbas": "https://lzblahblahblah.servicebus.windows.net/cromwell-runner-app-${workspaceId}/cbas",
+                        "cbas-ui": "https://lzblahblahblah.servicebus.windows.net/cromwell-runner-app-${workspaceId}/",
```

Comment:
Cromwell runner app won't have `cbas` or `cbas-ui`, and proxy url prefix will likely be either `cromwell-runner` or `cromwell-runner` (@salonishah11 would know better). Maybe doesn't affect the outcome of this test but probably good to keep our mock data consistent with the actual data

Reply:
I'll update for accuracy, and as far as outcome, it shouldn't matter too much, but once I know what it actually is, I can change that as well.

Reply:
I'll change this to cromwell-runner then for accuracy 👍

Reply:
Confirming that it will be changed to `cromwell-runner`.